src/EditorFeatures/CSharp/Simplification/CSharpGlobalIdeOptionsProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Simplification;
+
+namespace Microsoft.CodeAnalysis.CSharp.Simplification;
+
+[ExportLanguageService(typeof(IGlobalIdeOptionsProvider), LanguageNames.CSharp), Shared]
+internal sealed class CSharpGlobalIdeOptionsProvider : IGlobalIdeOptionsProvider
```

Comment:
what IGlobalIdeOptionsProvider and how does it compare to IGlobalOptionService?

Reply:
Oh, I think I added this before I added ISimplifierOptionsStorage, which makes this obsolete :)
Will remove.